bin/convert2vulyk.py

```diff
@@ -133,6 +142,8 @@ def _run_ner(txt: str) -> str:
                              '*.ann for lang-uk data set')
     parser.add_argument('--brat_annotation', type=argparse.FileType('r'), default=None,
                         help='Path to file with Brat standoff markup')
+    parser.add_argument('--drop_annotations', default=False, action="store_true",
```

Comment:
nit: 
1) I'd name it --dont_annotate as we are essentially preventing annotations to be created
I don't see the case when we'd read annotations with --brat_annotations and then drop them - doesn't seem to make much sense but I might be missing something

2) if 1st comment is true then it would make sense to make these parameters mutually exclusive. An example can be found here https://github.com/lang-uk/ner-uk/blob/e9e722dc51ab245c635c7aa90711856e11856b7d/scripts/convert_data.py#L244

Reply:
I gave it another look, in the light of trained Spacy model and decided to refactor that script in the following way:

- Add two subcommands, one to simply convert text (with optional .ann) to vulyk format, second to tag the raw text (using stanza/spacy and an option to specify the model).
- I'm still evaluating the idea of using the glob-like value instead of input file. The idea here is simple, most of the files will be converted in bulk, not one by one (and saved in jsonlines format rather than json). The only thing that stops me at the moment is optional annotation file(-s). E.g, if we asked to convert 10 txt files from the folder using /some/mask/*.txt, how do we provide annotations to all of them? One idea here is to simply substitute extension of the input file and make the end-user care about correct naming. I'll take another look tomorrow morning.


BTW, here is the results from Spacy.
<img width="1073" alt="image" src="https://user-images.githubusercontent.com/131186/149040053-0d5c0694-f948-4455-ac0a-7d44f4b050aa.png">
And Stanza
<img width="1213" alt="image" src="https://user-images.githubusercontent.com/131186/149040200-01478101-2db9-4291-b721-d7a5752a4d8c.png">

Reply:
These are great points.

On 1:
I like the idea of making default script behavior - as simple format converter - make vulyk format.

Providing --bratt_annotations would add annotations to output.

And a new optional 
--annotate_with_model (or smth similar) can be used to specify which model to use for annotations (spacy, stanza)

Reply:
On 2: glob parameter vs naming approach and autodiscovery

I'd prefer to assume naming scheme and auto discover annotation files that exist near by. I just often see those exist near by.
Unless our current need would require to transform dataset to match that format. 
In simple words let's just refactor to what we need now and not plan for some future generic case

Reply:
It's easier to convert the dataset, I think. Involved changes to the code is more complicate.

Reply:
On 1: I'll make subcommands, i.e ./convert2vulyk.py preannotate|convert.

I'll show you some code today, it'll be more descriptive, I think